roles/neutron-data-network/tasks/main.yml

```diff
@@ -59,6 +59,45 @@
     neutron-metadata-agent:
       config_files: /etc/neutron/metadata_agent.ini
 
+- name: neutron lbaas config
+  template:
+    src=etc/neutron/neutron_lbaas.conf
+    dest=/etc/neutron/neutron_lbaas.conf
+    mode=0644
+  notify:
+    - restart neutron services
```

Comment:
So we should have individual services for each of these daemons. We may want to restart that specific daemon as an operator task, or when the config for that daemon changes (but nothing else does). For that reason, I believe we should have a handler specifically for the lbaas restart.